packages/babel-plugin/src/utils/__tests__/Cache.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { Cache, CacheOptions } from '../Cache';
+
+jest.mock('@compiled/utils', () => {
+  return { ...jest.requireActual('@compiled/utils'), hash: () => 'hash-test' };
+});
+
+describe('#Cache', () => {
+  beforeAll(() => {
+    Cache.getUniqueKey = (key: string, namespace?: string) =>
+      namespace ? `${namespace}----${key}` : key;
+  });
+
+  const setup = (options: CacheOptions = {}) => {
+    const cache = new Cache(options);
+
+    return { cache };
```

Comment:
nit: are you expecting to add more to this return object? might as well just return it immediately :P

Reply:
Hehe. just convention I was following, if we have something more to return in future. Cool. Lets make the change 😄